includes/definitions/discovery/esphome.yaml

```diff
@@ -0,0 +1,79 @@
+mib_dir: esphome
+modules:
+    os:
+        hardware:
+            - ESPHOME-ESP8266-SNMP-MIB::chipModel.0
+            - ESPHOME-ESP8266-SNMP-MIB::chipRevision.0
+        hardware_template: 'Model: {{ $ESPHOME-ESP8266-SNMP-MIB::chipModel.0 }} Revision: {{ $ESPHOME-ESP8266-SNMP-MIB::chipRevision.0 }}'
+        version: SNMPv2-MIB::sysDescr.0
+        version_regex: '/(?<version>version [0-9]+\.[0-9]+\.[0-9]+)/'
```

Comment:
I would do this
```suggestion
        version_regex: '/version (?<version>[^ ]+)/'
```

Reply:
Before: 
![image](https://github.com/user-attachments/assets/61cc1a0f-06ed-48c7-a822-43bb9cb5f8d2)
After: 
![image](https://github.com/user-attachments/assets/cd4ce39d-9be2-44ed-a858-477318b1905f)

Looks good to me! I've committed the suggestion.